### PR TITLE
Added Dot(.) at the end of line.

### PR DESCRIPTION
**Pull request by ashishagarwal06**
Hi - Added Dot(.) to end the paragraph.

Thanks

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by ashishagarwal06** [comment]
Hi - Pls review and approve.

Thanks,
Ashish

File: docs/sources/administration/configuration.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ Directory where Grafana automatically scans and looks for plugins. Manually or a
 
 ### provisioning
 
-Folder that contains [provisioning]({{< relref "provisioning.md" >}}) config files that grafana will apply on startup. Dashboards will be reloaded when the json files changes
+Folder that contains [provisioning]({{< relref "provisioning.md" >}}) config files that grafana will apply on startup. Dashboards will be reloaded when the json files changes.
```
Comment (achatterjee-grafana) [inline docs/sources/administration/configuration.md:182]: Looks good. I have a request. Can we change grafana to Grafana. Thank you very much!

**Review by achatterjee-grafana — COMMENTED** [review]
Added a comment.

File: docs/sources/administration/configuration.md (hunk 1 above)
Comment (ashishagarwal06) [reply docs/sources/administration/configuration.md:182]: Yes, Pls..Thanks,
Regards,
Ashish

Comment (achatterjee-grafana) [reply docs/sources/administration/configuration.md:182]: Will you be kind and make the change? Thank you again!

Comment (ashishagarwal06) [reply docs/sources/administration/configuration.md:182]: Hi, I have raised separate pull request (28231) for the same.

Thanks,
Ashish

**Review by achatterjee-grafana — APPROVED** [review]

**Pull request merged**